foundation/src/main/java/org/researchstack/foundation/components/survey/ui/body/FormBody.java

```diff
@@ -23,12 +22,12 @@
     // Constructor Fields
     //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
     private FormStep step;
-    private StepResult<StepResult> result;
+    protected  StepResult<StepResult> result;
```

Comment:
Nit: What do you think about making `result` and `formStepChildren` package-private and marking them with `@VisibleForTesting`?